arm-datalake-store/filesystem/2016-11-01/swagger/filesystem.json

```diff
@@ -1290,59 +1425,65 @@
           "type": "integer",
           "readOnly": true,
           "format": "int64",
-          "description": "Gets the last access time as ticks since the epoch."
+          "description": "the last access time as ticks since the epoch."
         },
         "blockSize": {
           "type": "integer",
           "readOnly": true,
           "format": "int64",
-          "description": "Gets the block size for the file."
+          "description": "the block size for the file."
         },
         "childrenNum": {
           "type": "integer",
           "readOnly": true,
           "format": "int64",
-          "description": "Gets the number of children in the directory."
+          "description": "the number of children in the directory."
+        },
+        "expirationTime": {
+          "type": "integer",
+          "readOnly": true,
```

Comment:
It is only set using _SetExpiry which doesn't take this object in. I will make this a better description that indicates that if the value returned is 0, then there is no expiration.